STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.examiner.JavaSourceInspectorTest.testFindAndSetAttributesIgnoreCommentBlocks
MODIFIED FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/JavaSourceInspectorTest.java
CLEANER: de.strullerbaumann.visualee.examiner.TestGroup100Case0.test259
CLEANER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 15
NEW CLEANER SIZE: 2
ELAPSED TIME: 110.03099989891052
==========================
@@ -69,0 +69,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
